web/src/components/tasks/business-formation/business/BusinessStep.test.tsx

```diff
@@ -812,44 +812,62 @@ describe("Formation - BusinessStep", () => {
       const legalStructureId = "limited-liability-company";
       await getPageHelper({ legalStructureId }, {});
       const displayLegalStructure = screen.getByTestId("legal-structure");
-      expect(displayLegalStructure).toHaveTextContent(Config.formation.legalStructure[legalStructureId]);
+      expect(displayLegalStructure).toHaveTextContent(
+        Config.formation.legalStructure.domesticLabels[legalStructureId]
+      );
     });
 
     it("displays llp legal structure from profile data", async () => {
       const legalStructureId = "limited-liability-partnership";
       await getPageHelper({ legalStructureId }, {});
       const displayLegalStructure = screen.getByTestId("legal-structure");
-      expect(displayLegalStructure).toHaveTextContent(Config.formation.legalStructure[legalStructureId]);
+      expect(displayLegalStructure).toHaveTextContent(
+        Config.formation.legalStructure.domesticLabels[legalStructureId]
+      );
     });
 
     it("displays lp legal structure from profile data", async () => {
       const legalStructureId = "limited-partnership";
       await getPageHelper({ legalStructureId }, {});
       const displayLegalStructure = screen.getByTestId("legal-structure");
-      expect(displayLegalStructure).toHaveTextContent(Config.formation.legalStructure[legalStructureId]);
+      expect(displayLegalStructure).toHaveTextContent(
+        Config.formation.legalStructure.domesticLabels[legalStructureId]
+      );
     });
 
     it("displays sCorp legal structure from profile data", async () => {
       const legalStructureId = "s-corporation";
       await getPageHelper({ legalStructureId }, {});
       const displayLegalStructure = screen.getByTestId("legal-structure");
-      expect(displayLegalStructure).toHaveTextContent(Config.formation.legalStructure[legalStructureId]);
+      expect(displayLegalStructure).toHaveTextContent(
+        Config.formation.legalStructure.domesticLabels[legalStructureId]
+      );
     });
 
     it("displays cCorp legal structure from profile data", async () => {
       const legalStructureId = "c-corporation";
       await getPageHelper({ legalStructureId }, {});
       const displayLegalStructure = screen.getByTestId("legal-structure");
-      expect(displayLegalStructure).toHaveTextContent(Config.formation.legalStructure[legalStructureId]);
+      expect(displayLegalStructure).toHaveTextContent(
+        Config.formation.legalStructure.domesticLabels[legalStructureId]
+      );
     });
 
-    it("displays foreign label ahead of legal structure from profile data", async () => {
+    it("displays foreign label for legal structure for FOREIGN persona", async () => {
       const legalStructureId = randomPublicFilingLegalType();
```

Comment:
[dust] might be good to have a refactor ticket for gradually replacing random functions with it.each function for robustness. I don't think it would add much time to the CI tests (none in fact because I think the cypress stuff is still the limiting reactant) and I think coverage for tests in this regard is always good.